host-bmc/utils.cpp

```diff
@@ -56,6 +57,63 @@ Entities getParentEntites(const EntityAssociations& entityAssoc)
     return parents;
 }
 
+inline bool checkChassisDigits(const std::string& fullPath)
+{
+    std::regex chassisRegex(R"(/chassis\d{3,})");
+    if (!std::regex_search(fullPath, chassisRegex))
+    {
+        return false; // chassis has no digits
+    }
+    else
+    {
+        return true;
+    }
```

Comment:
this else can be simplified to return std::regex_search(fullPath, chassisRegex);